pkgs/by-name/ha/handheld-daemon/package.nix

```diff
@@ -1,35 +1,41 @@
 {
+  lib,
+  python3Packages,
   fetchFromGitHub,
+
+  # dependencies
   hidapi,
   kmod,
-  lib,
-  python3,
+  lsof,
   toybox,
-  lsof
 }:
-python3.pkgs.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "handheld-daemon";
-  version = "3.6.1";
+  version = "3.6.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "hhd-dev";
     repo = "hhd";
     rev = "refs/tags/v${version}";
-    hash = "sha256-IdpSRb66G+WzTv/BC29r2OjO1b4VdWbV6OSzOoiFAO0=";
+    hash = "sha256-W1Ap6yTryBDozKe3aO413Fu0RBul9kEA9ACUTdYyOKM=";
   };
 
-  propagatedBuildInputs = with python3.pkgs; [
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  dependencies = with python3Packages; [
     evdev
     hidapi
     kmod
+    lsof
+    pyserial
     pyyaml
     rich
     setuptools
     toybox
     xlib
```

Comment:
The dependency list here kind of mismatches https://github.com/hhd-dev/hhd/blob/v3.6.2/pyproject.toml#L19-L24 ?